docker/package.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+# SPDX-FileCopyrightText: 2022 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+import os
+import sys
+import shlex
+import shutil
+import subprocess
+from typing import Optional, List
+
+from package.package_generator import make_ubuntu_parser
+
+from build.util.build import parser
+from build.ubuntu.build import build_ubuntu
+from build.fedora.build import build_fedora
+
+from build.ubuntu.sign import sign_ubuntu
+from build.fedora.sign import sign_fedora
+
+import build.util.sign as sign
+from build.ubuntu.sign import sign_ubuntu
+from build.fedora.sign import sign_fedora
+
+import build.util.upload as upload
+from build.ubuntu.upload import upload_ubuntu
+from build.fedora.upload import upload_fedora
+
```

Comment:
I suppose that there's some issue with the imports here, because when I try to call this script (regardless of the arguments I pass) I get an error that ends with:
```
argparse.ArgumentError: argument --output-dir: conflicting option string: --output-dir
```

Reply:
Some quirk of `argparse`. Got resolved by deepcopying `parser`.